content/docs/install/manual_demo.md

```diff
@@ -101,13 +101,15 @@ state of allowed and blocked traffic between pods:
 | bookwarehouse |     ❌     |     ❌     |     ❌     |      \        |
 
 
-To show SMI Traffic Split, we will deploy an additional application:
+To show the SMI Traffic Split, you need to deploy an additional application:
 
-- `bookstore-v2` - this is the same container as the first `bookstore` we deployed, but for this demo we will assume that it is a new version of the app we need to upgrade to.
+- `bookstore-v2` - This is the same container as the first `bookstore` you deployed, but for this demo you can assume that it is a new version of the app you need to upgrade to.
 
 The `bookbuyer`, `bookthief`, `bookstore`, and `bookwarehouse` Pods will be in separate Kubernetes Namespaces with
 the same names. Each new Pod in the service mesh will be injected with an Envoy sidecar container.
 
+> Note: The following commands *must* be run in Bash and *not* in Powershell.
+
```

Comment:
As the commands will work in other unix shells (like zsh), I would rephrase this to say a Linux shell or compatible (as with WSL2).

Reply:
Done.